src/seabirdscientific/conversion.py

```diff
@@ -568,8 +570,8 @@ def convert_oxygen_to_umol_per_kg(ox_values: np.ndarray, potential_density: np.n
     :return: oxygen values converted to milligrams/Liter
     """
     # From the SBE43 and SBE63 manual
```

Comment:
When referring to documents, it might be a good idea to include which version of the manual is being referenced. Customers will occasionally have outdated copies of some docs.